static/js/snap-pages/components/PublishedSnapSection/PublishedSnapSection.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import { ISnap } from "../../types";
+import { PublishedSnapList } from "../PublishedSnapList";
+import { PublisherMetrics } from "../PublisherMetrics";
+import { PAGE_NUMBER } from "../../types/constants";
+
+export const PublishedSnapSection = ({
```

Comment:
I think this should be a named function and the export at the bottom of the file to match other React components